prow/plugins/ownersconfig/config.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ownersconfig
+
+// Filenames configures which file names should be used for the OWNERS and OWNERS_ALIASES
+// concepts for a repo, if it's not the default set.
+type Filenames struct {
+	Owners        string `json:"owners,omitempty"`
+	OwnersAliases string `json:"owners_aliases,omitempty"`
+}
+
+const (
+	DefaultOwnersFile        = "OWNERS"
+	DefaultOwnersAliasesFile = "OWNERS_ALIASES"
+)
+
+type Resolver func(org, repo string) Filenames
+
+// FakeResolver fills in for tests that use a resolver but aren't testing it.
+func FakeResolver(_, _ string) Filenames {
+	return FakeFilenames
+}
+
+// FakeFilenames fills in for tests that need a Filenames but aren't testing them.
+var FakeFilenames = Filenames{
```

Comment:
Can we rename this to `DefaultFileNames`? The current naming is a bit confusing, as it makes for example tests look like they use a non-default set of names

Reply:
I explicitly didn't name it that way since I do *not* want production code to use these, they exist explicitly for testcases.

Reply:
I can make that more explicit in the doc.

Reply:
Sure, but that doesn't change anything about the fact that they are the default and not a `Fake`